CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0 (September 25, 2020)
```

Comment:
But this versions isn't upgraded in [package.json](https://github.com/elasticio/rest-api-component/pull/149/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2)..